client/state/data-layer/wpcom/me/blocks/sites/index.js

```diff
@@ -0,0 +1,46 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { READER_SITE_BLOCKS_REQUEST, READER_SITE_BLOCKS_RECEIVE } from 'state/action-types';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { registerHandlers } from 'state/data-layer/handler-registry';
+
+export const handleSiteBlocksRequest = action =>
+	http(
+		{
+			apiNamespace: 'wpcom/v2',
+			method: 'GET',
+			path: '/me/blocks/sites',
+			query: {
+				page: action.page || null,
+				per_page: action.perPage || null,
+			},
+		},
+		action
+	);
+
+export const siteBlocksRequestReceived = ( action, apiResponse ) => ( {
```

Comment:
Does it make sense to rename `apiResponse` to `payload`? This way we'd be able to shorten:

`payload: apiResponse,`

to

`payload,`